### PR TITLE
Added shortcut of 'H' during solving

### PR DESCRIPTION
**Pull request by Saurabh3333**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -354,6 +354,21 @@ module.exports = function () {
 
         break;
 
+      case 'h':
+
+        console.log("Keyboard shortcuts:");
```
Comment (icyflame) [inline index.js]: - Please abstract this out to another function (say `print_help`).
- Print the help only `if (!solving && !inspecting && !post_inspecting)`

**Review by icyflame — CHANGES_REQUESTED** [review]

File: index.js [hunk 2]
```diff
@@ -354,6 +368,14 @@ module.exports = function () {
 
         break;
 
+      case 'h':
+
+        if(!solving && !inspecting && !post_inspecting) {
+        	print_help();
```
Comment (icyflame) [inline index.js]: fix indentation

**Review by icyflame — CHANGES_REQUESTED** [review]
- Call `print_help` from the other location these things are printed as well
- Install editor config to use the appropriate indent

File: printhelp-module.js [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+methods.update_start_inspect = function (start_inspect) {
+  return start_inspect + 8;
+};
+
+methods.update_start_solve = function (start_solve) {
+  return start_solve + 8;
+};
+
+methods.print_help  = function (start_inspect) {
+  console.log("Keyboard shortcuts:");
```
Comment (icyflame) [inline printhelp-module.js]: This function should print some lines on the right column and some on the left. See [this](https://github.com/icyflame/cli-cube-timer/blob/83ae23472c31b1fb6143acd17804719e621b5c5e/index.js#L377-L390)

**Review by icyflame — CHANGES_REQUESTED** [review]
Hey, I think you misunderstood my last review. I told you to call the function `print_help` from [this part](https://github.com/icyflame/cli-cube-timer/blob/83ae23472c31b1fb6143acd17804719e621b5c5e/index.js#L377-L390) of the module. 

_Not_ to move the function to another js file. That's not desirable.

File: printhelp-module.js (hunk 3 above)
Comment (Saurabh3333) [reply printhelp-module.js]: Sir, should I leave the new js file and call function from the js whenever there is need to print the help message?
`This function should print some lines on the right column and some on the left.` can you please elaborate it as I am little confused?

Comment (icyflame) [reply printhelp-module.js]: 1. No, I would prefer not to complicate things further and keep everything in the same file. Put `print_help` in `index.js` and call it wherever you need to print the Help text.

2. If you see this block of code: https://github.com/icyflame/cli-cube-timer/blob/83ae23472c31b1fb6143acd17804719e621b5c5e/index.js#L377-L390

You will notice that the cursor is repositioned using `charm` before printing using `console.log`. So, the `print_help` function must also do this.

**Review by icyflame — APPROVED** [review]

**Comment by Saurabh3333** [comment]
Should I close this pull request and create new for the new issue or continue working here?

**Comment by icyflame** [comment]
Hey @Saurabh3333, I will try to merge this in tomorrow. Meanwhile, you can open a new PR. But I must warn you that once I merge this code in, there might be some merge conflicts that might come up which you will have to fix.

**Comment by Saurabh3333** [comment]
That would be a learning experience, sir.

**Comment by icyflame** [comment]
Hey @Saurabh3333, thanks a lot for your patch. I have merged it now with some changes. Specifically, I have changed the structure of the code to make it easily extensible. See the final `print_help` function: https://github.com/icyflame/cli-cube-timer/blob/1c0a94a8bd09be3365dd910a73b874d66228bc11/index.js#L133-L151

I wanted to add the last line (referring to `H` as the Help shortcut) When I went to do this, I realized that the existing structure in this patch, would mean adding the line, adding some code around the line and then incrementing some values across `index.js`. That chain of thought led me to this enhancement. Adding a line here will immediately fix the UI everywhere.

I also realized that `start_solve` is redundant (#50). After we can work together to merge #49, you can take a shot at fixing 50 as well. :slightly_smiling_face:

**Comment by Saurabh3333** [comment]
Sure sir.

**Pull request merged**